target/linux/ath79/dts/qca9563_compex_wpj563.dts

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "Compex WPJ563";
+	compatible = "compex,wpj563", "qca,qca9563";
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		sig1 {
+			label = "wpj563:green:sig1";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
ar71xx sets up this led for diag. So, you could use it via the aliases node (or another one of the sigX leds dependent on which makes sense) ...

Reply:
i've chosen sig4 to be consistent with the wpj531 and because i've noticed that sometimes sig1 is triggered by something else (don't know what).

Reply:
haven't had sig1 do anything for a while now.
probably a mistake on my end from when i tested the numerous different vendor firmwares.


in (openwrt) ar71xx, the choice for the status LED is not consistent: wpj342, wpj531, and wpj558 use sig3; wpj563 uses sig1 .
with the compex firmware, all LEDs are used for rssiled + sig4 for "diag" (except for the wpj342).
sig4 is used by u-boot and is active when booting the stock firmware.
it's documented as 'RSSI4/Diag' in the hardware guides for WPJ344/WPJ531/WPJ563.

i'd say sig4 is the best choice for the status LED for consistency with the stock-image, the official documentation, and the preinstalled u-boot.